databuilder/databuilder/task/search/search_metadata_to_elasticsearch_task.py

```diff
@@ -153,6 +152,8 @@ def run(self) -> None:
             # create index
             LOGGER.info(f"Creating ES index {self.elasticsearch_new_index}")
             index = Index(name=self.elasticsearch_new_index, using=self.elasticsearch_client)
+            doc_mapping = self.document_mapping._doc_type.mapping.meta(name='_meta', params={'new_mapping': True})
```

Comment:
nit: how about `params={'version': 2}`

Reply:
this is better for iteration you are right